R/annotation-logticks.r

```diff
@@ -126,13 +126,15 @@ GeomLogticks <- ggproto("GeomLogticks", Geom,
                         mid = unit(0.2, "cm"), long = unit(0.3, "cm"))
   {
     ticks <- list()
+    flipped <- inherits(coord, "CoordFlip")
```

Comment:
This feels a bit too specific to this problem. Are there any other way to check if the coord is flipped? (I guess the answer is no, but let me just confirm...)

Reply:
The answer is no :-)... We have these annoying specific logic pertaining to coord_flip around the code base and there is not really much to do about it at this time

We can discuss a more throughout fix at a later stage but that is beyond this fix

Reply:
I see, thanks for clarifying.